.github/workflows/helm.yml

```diff
@@ -16,6 +16,16 @@ on:
         description: Repository {{ github.repository }} <owner>/<repository>
         default: ${{ github.repository }}
         required: false
+      force-publish:
```

Comment:
I feel like `force-publish` is not really appropriate here. What do you think of `is-pre-release` (even if verbose...)?

Reply:
Mmmmh alpha releases are not exactlty pre-releases